### DISCUSSION TITLE
Possible to include a UI feature to display the Refresh Rate when changed from 60 Hz to 240 Hz and vice versa?

### DISCUSSION THREAD

**Discussion by prabhatnagpal** [BartoszCichecki/LenovoLegionToolkit#1398] [Ideas]
Just like during the use of Vantage services, when we remove the AC adapter from charging, it changes the refresh rate from 240 Hz to 60 Hz, and also displays 60 Hz on screen, and vice versa.
Can the UI feature be added in Lenovo Legion Toolkit also, such that when refresh rate change occurs, the refresh rate is displayed on screen also?
![image](https://github.com/user-attachments/assets/df97e388-4ee9-4162-823c-ec8c11a32e3b)
![image](https://github.com/user-attachments/assets/1d9088a3-aec8-4b5d-8b51-173f4adb5887)

**Comment by BartoszCichecki** [comment]
A notification like this will be shown when you use Fn+R. Otherwise if you want to configure refresh rate switching you would use Actions to achieve the same. You can add custom notification text too. But I am not sure why that would be necessary.

**Reply by prabhatnagpal** [reply]
When I had used Lenovo Vantage before shifting to Lenovo Legion Toolkit, every time I used to remove the power adapter, it used to show the 60 Hz display icon on screen, and when I used to plug it back in it used to show the 240 Hz icon. This thing is not necessary but I was just thinking as per UI perspective.
I have already done what you have mentioned, that is changing of Refresh Rate by Actions tab, was just hoping for OSD (icon display for refresh rate) feature from Lenovo Vantage in Lenovo Legion Toolkit.

**Reply by prabhatnagpal** [reply]
"Show Notification" Step in actions tab is also fine, I just added that to display 240 Hz/60 Hz when power adapter is plugged/unplugged.
